doc/src/stdlib/arrays.md

```diff
@@ -74,6 +74,7 @@ Base.flipdim
 Base.circshift
 Base.circshift!
 Base.circcopy!
+Base.contains(eq::Function, itr, x)
```

Comment:
@MichaelHatherly is naming the input args (instead of just their types) useful / a good idea here?